### PR TITLE
feat: use new MFA by translating the query payload

### PR DESCRIPTION
**Pull request by standarius**
[Related task](https://app.asana.com/0/1199914526147392/1200265329674322/f)

This PR is alternative to https://github.com/standardnotes/auth/pull/21

### CODE REVIEW

File: src/Controller/AuthController.ts [hunk 1]
```diff
@@ -63,8 +63,8 @@ export class AuthController extends BaseHttpController {
     }
 
     const verifyMFAResponse = await this.verifyMFA.execute({
-      email: <string> request.query.email,
-      token: <string> request.query.mfa_key,
+      email: request.query.email as string,
+      token: request.query.enteredTotp as string,
```
Comment (karolsojko) [inline src/Controller/AuthController.ts:67]: hmm won't this break currently existing clients?

Comment (standarius) [reply src/Controller/AuthController.ts:67]: Right, this should be released as Auth 2.0.0. Forgot to include the change to package.json.

Comment (standarius) [reply src/Controller/AuthController.ts:67]: updated

Comment (karolsojko) [reply src/Controller/AuthController.ts:67]: change in `package.json` are irrelevant to be honest on our "application projects" - they only matter on the `snjs` monorepo where we have packages.

on applications like this one every merge to `develop` gets published to DEV environment and every change merged to `main` branch gets published to PRODUCTION environment

Comment (standarius) [reply src/Controller/AuthController.ts:67]: so how do we handle breaking changes like this?

Comment (standarius) [reply src/Controller/AuthController.ts:67]: (discussion moved to the ticket)

**Comment by standarius** [comment]
dropping the change and closing as explained in [the ticket](https://app.asana.com/0/1199914526147392/1200265329674322/f)

**Pull request closed without merging**